3.0/README.md

```diff
@@ -154,6 +154,55 @@ An array of tile endpoints. {z}, {x} and {y}, if present, are replaced with the
 ```
 
 ## 3.15 `vector_layers`
+
+REQUIRED. (if describing vector tiles)
+
+A JSON object whose value is an array of JSON objects. Each of those JSON objects describes one layer of vector tile data, and MUST contain the following key-value pairs:
+
+* `id` (string): The layer ID, which is referred to as the `name` of the layer in the [Mapbox Vector Tile spec](https://github.com/mapbox/vector-tile-spec/tree/master/2.1#41-layers).
```

Comment:
It's not clear to me what the reference to the MVT spec is.

Reply:
Per discussion in #14 it is to help connect what this value refers to according to the vector tile layer concept. The MVT spec is used as an example. If we remove this reference entirely, it could be confusing since it’s nice to point to how vector tile specifications refer to layer names. I think we can remove the reference in v4 once we rename the property to `name`. 

Basically, if we're considering matching the `id` field to `name` to match the MVT specification, then the reference link to the MVT specification feels relevant.